src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -107,41 +107,19 @@ export default function FormNav(props) {
         setIndex(index - 1);
       }
 
-      return () => {
-        // Check main toggle to enable custom focus; the unmounting of the page
-        // before the review & submit page may cause the customScrollAndFocus
-        // function to be called inadvertently
-        if (
-          !(
-            page.chapterKey === 'review' ||
-            window.location.pathname.endsWith('review-and-submit')
-          )
-        ) {
-          if (
```

Comment:
I remembered one of the unintentional side effects of the old code being in the return was that it basically acted as a setTimout. 
So now that it relies on FormPage which is immediate, one of the bugs I'm seeing right now is when I go to the next page, it will focus correctly, but the page title is still the previous page title for a brief moment and the screen reader reads out the previous pages value when you go to the next page. I think a setTimeout of 0 wrapper the focus logic in the FormPage might fix this.